index.html

```diff
@@ -90,7 +97,7 @@ <h5>CANOPY</h5>
               <li class="btn1">css</li>
               <li class="btn1">Javascript</li>
             </ul>
```

Comment:
- [ ] Remove all code related to project information from the HTML structure. All project details should be displayed dynamically using javascript code.